### ISSUE TITLE
Feature Request: Pixel Analyzer

### ISSUE THREAD

**Issue by ahemberger** [NatronGitHub/Natron#320]
Hi there! I’d like to migrate fully from Nuke to Natron for personal projects, but a critical utility that Nuke offers that’s missing from Natron (or, at least, I can’t seem to find it) is the Pixel Analyzer panel. It’s super-useful for color correction and white balancing stuff. Just wanted to throw a vote out for it to be included in a future release.

**Comment by jeinstei** [comment]
Is this treated as a node with inputs, etc? I've never used Nuke

**Comment by ahemberger** [comment]
No, it’s an informational panel in Nuke.

**Comment by threedslider** [comment]
It needs some screenshot to get idea ? I don't own Nuke.

**Comment by ahemberger** [comment]
The Foundry offer a free, Non-commercial version of Nuke that is freely available for people to download and use. 

The Pixel Analyzer is a palette that opens like any other properties pane; it is not associated with a specific compositing node, but rather offers meaningful information about sampled pixel(s) from the Viewer:

**Comment by ahemberger** [comment]
The Foundry offer a free, Non-commercial version of Nuke that is freely available for people to download and use. 

The Pixel Analyzer is a palette that opens like any other properties pane; it is not associated with a specific compositing node, but rather offers meaningful information about sampled pixel(s) from the Viewer:

<img width="612" alt="screen shot 2019-02-28 at 9 43 06 am" src="https://user-images.githubusercontent.com/21340636/53578876-20741380-3b3e-11e9-8394-df2c69f88c55.png">

**Comment by threedslider** [comment]
Yeah it is a feature that can make it possible but right now we need developer or maintainer... I am somewhat a contributor and not yet brighting some feature to Natron but it is coming for 3D camera. All i can say you need to wait and you will get it in some way here later.

**Comment by devernay** [comment]
pixel analyzer appeared in Nuke 8.
I agree that would be a nice feature to have, but it requires some work.
In the mean time, you can use the ImageStatistics plugin (which samples a single rectangular area)

**Comment by threedslider** [comment]
It is possible to integrate in it so sure it needs some work...

Possibly to my todo as well.
